lib/msf/ui/console/command_dispatcher/core.rb

```diff
@@ -1860,7 +1860,7 @@ def get_matching_sessions(search_term)
       when LAST_CHECKIN
         checkin_searches << term
       else
-        print_error("Please provide valid search term. Given: #{term.split(':').first}")
+        print_error("Please provide valid search term. Given: #{term.split(':').first}. Supported keywords are: session_id, session_type, last_checkin")
```

Comment:
This isn't super future proofed, i.e. if we ever add additional keywords in the future here - we're most likely going to forget to update this documentation

Can we generate this list from Ruby instead? 👀